ci/infra/openstack/cloud-provider.tf

```diff
@@ -14,6 +14,6 @@ resource "null_resource" "generate_cloud_provider_conf" {
       OS_PUBLIC_NET_ID     = "${data.openstack_networking_network_v2.external_network.id}"
     }
 
-    command = "bash generate-cloud-provider-conf"
+    command = "bash generate-cpi-conf.sh"
   }
```

Comment:
Sorry I don't get this. Only changing this makes generate-cpi-conf.sh file be in the package?

Reply:
yes, this is because
```
for file in ci/infra/openstack/*.*; do
  install -p -m 0644 $file %{buildroot}/%{_datadir}/caasp/terraform/openstack/
done
```
so it means that only files `*.*` will be installed